src/reference/asciidoc/amqp.adoc

```diff
@@ -191,8 +192,12 @@ _Optional_.
 
 <25> Tells the `SimpleMessageListenerContainer` how many messages to process in a single transaction (if the channel is transactional).
 For best results it should be less than or equal to the set "prefetch-count".
+Not allowed when 'consumers-per-queue' is set.
 _Optional (Defaults to 1)_.
 
+<26> Indicates that the underlying listener container should be a `DirectMessageListenerContainer` instead of the default `SimpleMessageListenerContainer`.
+Refer to the Spring AMQP Reference Manual for more information.
```

Comment:
OK. Does this work for you: http://docs.spring.io/spring-amqp/reference/html/_reference.html#_container ?
When S-A 2.0 is GA, the symlink will point to the proper documentation.
Although we may leave with your current expression.
We have a TIP above on the matter:
```
Please familiarize yourself with the
http://docs.spring.io/spring-amqp/reference/html/[reference documentation of the Spring AMQP project as well].
```